<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">

 <class>PasswordDialog</class>

 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>

  <property name="windowTitle">
   <string>Password</string>
  </property>

  <layout class="QVBoxLayout">

   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter password:</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLineEdit" name="confirmEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm password</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLabel" name="errorLabel">
     <property name="styleSheet">
      <string>color: red;</string>
     </property>
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>

     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections/>

</ui>
